copy_cat/settings/common.py

```diff
@@ -21,7 +21,8 @@
     # Local
     'base',
     'forum',
-    'user'
+    'user', 
+    "log_reg"
```

Comment:
`log_reg` isn't a good module name. You'd be better off calling it `registration`.